.github/workflows/ray-docker.yml

```diff
@@ -8,44 +8,26 @@ jobs:
         with:
           fetch-depth: 1
       - shell: bash -l {0}
-        run: bash examples/docker/ci-ray-docker/build-docker-image.sh
-      - shell: bash -l {0}
-        run: docker create --name ray_container ray_docker_image:1.1.0 bash -c "tail -f /dev/null"
-      - shell: bash -l {0}
-        run: docker start ray_container
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/experimental/xgboost/test/test_default.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_binary.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_default.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_indexing.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_iter.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_join_sort.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_map_metadata.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_reduction.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_udf.py
-      - shell: bash -l {0}
-        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_window.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_series.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_rolling.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_concat.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_groupby.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_reshape.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_general.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/pandas/test/test_io.py
-      - shell: bash -l {0}
-        run: docker exec ray_container python -m pytest modin/experimental/pandas/test/test_io_exp.py
+        run: docker build -f examples/docker/ci-ray-docker/ci-ray-docker.dockerfile -t ray_docker_image:1.1.0 .
```

Comment:
is there a reason for CI dockerfile to live under `examples`?..

Reply:
There is no actual reason for this, i bewared that placing dockerfile inside `.github/workflows` directory can corrupt Github Actions, but looks like CI works fine in that case.